action.yml

```diff
@@ -20,10 +20,52 @@ branding:
 runs:
   using: composite
   steps:
+    # Set up a non-EOL, cibuildwheel & pipx supported Python version
```

Comment:
This could be reworked as a helper script with its own tests.
Syntax is Python 3.4+
Unfortunately, setup-python messes with the environment for other steps & I'd rather avoid side-effects for the rest of users workflow when using the cibuildwheel GitHub Action.
It should allow self-hosted runners to continue working out-of-the-box.